CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Make this > 3.10, since it's started to give deprecation warnings on Humble

Reply:
No constructs from later cmake are used, but I guess this needs to be updated in templates used for
`ros2 pkg create `
which currently puts 3.8 in for Humble.
I will update to 3.16

Reply:
Good point, I just opened https://github.com/ros2/ros2cli/pull/969